src/sage/combinat/symmetric_group_representations.py

```diff
@@ -999,6 +1001,186 @@ def _repr_(self):
         """
         return "Specht representations of the symmetric group of order %s! over %s" % (self._n, self._ring)
 
+# #### Unitary Representation ###############################################
+
+
+class UnitaryRepresentation(SymmetricGroupRepresentation_generic_class):
+    r"""
+    This is a unitary representation of the symmetric group.
+    In characteristic zero, this is the same as the orthogonal representation since
```

Comment:
```suggestion
    r"""
    A unitary representation of the symmetric group.

    In characteristic zero, this is the same as the orthogonal representation since
```
Thanks. Here is one formatting change.